app/decorators/user_decorator.rb

```diff
@@ -12,4 +12,8 @@ def display_role
       role.humanize.capitalize
     end
   end
+
+  def display_company_name
```

Comment:
Краще в моделі юзера напиши `delegate :name, to: :company, prefix: true`

Reply:
Але краще в декораторі - наразі це логіка, що потрібна для відображення, відповідно вона належить до view леєра.
```suggestion
  def company_name
```